src/routes/comments.js

```diff
@@ -0,0 +1,122 @@
+const express = require("express");
+const router = express.Router();
+const { body, validationResult } = require("express-validator");
+const Comment = require("../models/Comment");
+const User = require("../models/User");
+const mongoose = require("mongoose");
+
+//Comment Routes
+
+/*
+Route GET requests at root 
+return response containing all comments in JSON on success
+*/
+router.get("/", function(req, res) {
+  Comment.find()
+    .lean()
+    .exec(function(err, comments) {
```

Comment:
I think you should do the populate here and get the username. Of course, you can do that later as well.

Reply:
Great, I went ahead and moved it here.